src/textual/command.py

```diff
@@ -1,9 +1,13 @@
-"""The Textual command palette."""
+"""The Textual command palette.
+
+See the guide on the [Command Palette](../guide/command_palette.md) for full details.
+
```

Comment:
```suggestion
```

Reply:
I don't think we should be too fussed about additional lines in docstrings.

Reply:
I wasn't going to make a PR to delete them, but this was one click away 🤷